kube-shell/Dockerfile

```diff
@@ -2,29 +2,23 @@ ARG ALPINE=alpine:latest
 FROM ${ALPINE} as alpine
 ARG ARCH
 
+ARG KUBECTL_VERSION=v1.29.0
+ARG HELM_VERSION=v3.13.3
+
 RUN apk add -U --no-cache bash bash-completion curl jq
```

Comment:
how come the same changes still showing up after merging this https://github.com/portainer/docker-images/pull/14/files